resources/sts/4.14/hypershift/sts_extended_hcp_support_permission_policy.json

```diff
@@ -2,58 +2,133 @@
+                "ec2:GetSerialConsoleAccessStatus"
+            ],
+            "Resource": "*"
+        },
+        {
+            "Sid": "DescribeNetworkInsightsPath",
+            "Effect": "Allow",
+            "Action": [
+                "ec2:DescribeNetworkInsightsPaths",
+                "ec2:StartNetworkInsightsAnalysis"
+            ],
+            "Resource": "*"
+        },
+        {
+            "Sid": "NetworkInsightsPath",
+            "Effect": "Allow",
+            "Action": [
+                "ec2:DeleteNetworkInsightsPath"
+            ],
+            "Resource": "arn:aws:ec2:*:*:network-insights-path/*"
+        },
+        {
+            "Effect": "Allow",
+            "Action": [
                 "ec2:CreateNetworkInsightsPath",
-                "ec2:CreateSnapshot",
-                "ec2:CreateSnapshots",
-                "ec2:DeleteNetworkInsightsAnalysis",
-                "ec2:DeleteNetworkInsightsPath",
-                "ec2:DescribeAccountAttributes",
-                "ec2:DescribeAddresses",
-                "ec2:DescribeAddressesAttribute",
-                "ec2:DescribeAggregateIdFormat",
-                "ec2:DescribeByoipCidrs",
-                "ec2:DescribeCapacityReservations",
-                "ec2:DescribeCarrierGateways",
-                "ec2:DescribeClassicLinkInstances",
+                "ec2:DescribeNetworkInsightsAnalyses"
+            ],
+            "Resource": "*"
+        },
+        {
+            "Sid": "NetworkInsightsAnalysis",
+            "Effect": "Allow",
+            "Action": [
+                "ec2:DeleteNetworkInsightsAnalysis"
+            ],
+            "Resource": "arn:aws:ec2:*:*:network-insights-analysis/*"
+        },
+        {
+            "Sid": "ClientVpn",
+            "Effect": "Allow",
+            "Action": [
                 "ec2:DescribeClientVpnAuthorizationRules",
                 "ec2:DescribeClientVpnConnections",
-                "ec2:DescribeClientVpnEndpoints",
                 "ec2:DescribeClientVpnRoutes",
-                "ec2:DescribeClientVpnTargetNetworks",
-                "ec2:DescribeCoipPools",
-                "ec2:DescribeCustomerGateways",
-                "ec2:DescribeEgressOnlyInternetGateways",
-                "ec2:DescribeIdentityIdFormat",
-                "ec2:DescribeIdFormat",
-                "ec2:DescribeImageAttribute",
-                "ec2:DescribeImages",
+                "ec2:DescribeClientVpnTargetNetworks"
+            ],
+            "Resource": "arn:aws:ec2:*:*:client-vpn-endpoint/*"
+        },
+        {
+            "Effect": "Allow",
+            "Action": [
+                "ec2:DescribeClientVpnEndpoints"
+            ],
+            "Resource": "*"
+        },
+        {
+
+            "Sid": "Vpn",
+            "Effect": "Allow",
+            "Action": [
+                "ec2:DescribeVpnConnections",
+                "ec2:DescribeVpnGateways"
+            ],
+            "Resource": "*"
```

Comment:
Can these actions be restricted? Do they need to be broken out?

Reply:
their condition keys resources are the same so probably can be grouped
`
	Grants permission to describe one or more VPN connections	List		
ec2:Region

DescribeVpnGateways	Grants permission to describe one or more virtual private gateways	List		
ec2:Region
`